js/views/PageNav.js

```diff
@@ -36,9 +36,8 @@ export default class extends View {
 
     $(document).on('click', this.onDocClick.bind(this));
 
-    this.listenTo(app.localSettings, 'change:macStyleWinControls',
-      this.onWinControlsStyleChange);
-    this.setWinControlsStyle(app.localSettings.get('macStyleWinControls') ? 'mac' : 'win');
+    app.localSettings.on( 'change:windowControlStyle', (_,style) => this.setWinControlsStyle(style));
```

Comment:
We should be using `listenTo` here, not `on`. `listenTo` gives you the added benefit that when this view is removed, the handler will be unbound, this preventing our removed view remaining as a ghost view because a bound handler has a reference to it.